.buildkite/premerge.steps.yaml

```diff
@@ -0,0 +1,14 @@
+---
+steps:
+  # Run the UnrealGDK premerge with this version of the TestGyms repo
+  # only run for PRs
+  - label: "unreal-gdk-premerge"
+    if: build.pull_request.id != null || build.source == "ui" || build.branch == "master" || build.branch == "release" || build.branch == "preview" || build.branch =~ /-rc/
```

Comment:
is this if doing what the comment says? My limited understanding of this is that this will cause any PRs to run as well as any changes to master/release/preview etc.

Reply:
Yeah, the comment is a bit outdated. PRs are run, changes to "big" branches are run.